### PR TITLE
Added Launch FAQs

### PR DESCRIPTION
**Pull request by tristan-spaulding2**
## Description

What does the pull request do? If it fixes a bug or resolves a feature request, be sure to link to that issue.

## Ticket

Does this PR fix an existing issue? If yes, provide a link to the ticket here:

## Checklist

Check if your PR fulfills the following requirements. Put an `X` in the boxes that apply.

- [ ] Files I edited were previewed on my local development server with `yarn start`. My changes did not break the local preview.
- [ ] Build (`yarn docusaurus build`) was run locally and successfully without errors or warnings.
- [ ] I merged the latest changes from `main` into my feature branch before submitting this PR.

### CODE REVIEW

**Comment by tristan-spaulding2** [comment]
@KyleGoyette I put subset of the FAQs in here, can add more later

File: docs/guides/launch/launch-faqs.md [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+
+  ```bash
+  wandb launch --help
+  ```
+
+### Can Launch automatically provision (and spin down) compute resources for me in the target environment?
+
+No. Launch uses existing resources you have created.  However, the Solution Architects at W&B are happy to work with you to configure your underlying Kubernetes infrastructure to facilitate retries, autoscaling, and use of spot instance node pools.  Reach out to [support@wandb.com](support@wandb.com) or your shared Slack channel.
+
+### `wandb launch -d` uploading a whole docker artifact and is not pulling from a registry?
```
Comment (ngrayluna) [inline docs/guides/launch/launch-faqs.md]: Confused by the wording, question.

Comment (tristan-spaulding2) [reply docs/guides/launch/launch-faqs.md]: fixed

**Review by ngrayluna — APPROVED** [review]

File: docs/guides/launch/launch-faqs.md [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+
+This will build a job when you create a run.
+
+Alternatively, you can make a launch job based on a docker image. See the [Create a job](https://docs.wandb.ai/guides/launch/create-job) page for more information. 
+
+### Are there best practices for using Launch effectively?
+
+  1. Create your queue before you start your agent, so that you can set your agent to point to it easily.  If you don’t do this, your agent will give errors and not work until you add a queue.
+  2. Create a W&B service account to start up the agent, so that it's not tied to an individual user account.
+  3. Use `wandb.config` to read and write your hyperparameters, as opposed to argsparse, so that they can be overwritten when re-running a job.  Check out [this guide](https://docs.wandb.ai/guides/launch/create-job#making-your-code-job-friendly) if you use argsparse.
```
Comment (KyleGoyette) [inline docs/guides/launch/launch-faqs.md]: I'd drop the "as opposed to argparse" here, we actually suggest putting the argparse obj into the config.

**Review by KyleGoyette — APPROVED** [review]
One nit, but LGTM generally

File: docs/guides/launch/launch-faqs.md (hunk 2 above)
Comment (tristan-spaulding2) [reply docs/guides/launch/launch-faqs.md]: fixed

**Pull request merged**